doc/website/blog/2022-08-22-atlas-ci-github-actions.md

```diff
@@ -0,0 +1,160 @@
+cautious when it comes to making changes to their databases. In fact, many teams set explicit
+policies on what kinds of changes to the database are allowed, often completely prohibiting
+any change that is destructive.
+
+Destructive changes are changes to a database schema that result in loss of data. For instance,
+consider a statement such as:
+
+```sql
+ALTER TABLE `users` DROP COLUMN `email_address`;
+```
+
+This statement is considered destructive because whatever data is stored in the `email_address`
+column will be deleted from disk, with no way to recover it.
+
+Suppose you were in charge of a team that decided to prohibit destructive changes,
+how would you go about enforcing such a policy? From our experience, most teams enforce
+policies relating to schema migrations in code-review: a human engineer, preferably with
+some expertise in operating databases, manually reviews any proposed database migration
+scripts and rejects them if they contain destructive changes.
+
+Relying on a human reviewer to enforce such a policy is both expensive (it takes time and mental energy)
+and error-prone.  Just like manual QA is slowly being replaced with automated testing, and manual
+code style reviews are being replaced with linters, isn't it time that we automate the process
+of ensuring that changes to database schemas are safe?
+
+### Announcing the Atlas GitHub Action
+
+Today, we're happy to announce the release of the official [Atlas GitHub Action](https://github.com/ariga/atlas-action)
+which can be used to apply [migration directory linting](2022-07-14-announcing-atlas-lint.md) for a bunch of popular
+database migration tools. [golang-migrate](https://github.com/golang-migrate/migrate),
+[goose](https://github.com/pressly/goose), [dbmate](https://github.com/amacneil/dbmate) and Atlas itself are already supported,
+and Flyway and Liquibase are coming soon.
+
+If you're using GitHub to manage your source code, you're in luck. By adding a short YAML
+file to your repository, you can start linting your schema migration scripts today! Let's see a
+short example.
+
+### Setting up
+
+Suppose we are running a website for an e-commerce business. To store the data for our website
+we use a MySQL database. Because the data in this databaseis _everything_ to us, we use
+a careful _versioned migrations_ approach where each change to the database schema is
+described in an SQL script and stored in our Git repository. To execute these scripts we use
+a popular tool called [golang-migrate](https://github.com/golang-migrate/migrate).
+
+The source code for this example can be found in [rotemtam/atlas-action-demo](https://github.com/rotemtam/atlas-action-demo/tree/master/migrations).
+
+Initially our schema contains two tables: `users` and `orders`, documented in the first few migration files:
+
+Create the `users` table:
+```sql title=migrations/20220819060736.up.sql
```

Comment:
I think golang-migrate requires a name for the migration files: `<version>_initial.up.sql` and similar. Therfore, executing the files in the repo with golang-migrate wont work.

Reply:
<img src="https://media1.giphy.com/media/3o7TKqnN349PBUtGFO/giphy.gif"/>